crates/sui-types/src/effects/effects_v1.rs

```diff
@@ -1,20 +1,21 @@
 // Copyright (c) Mysten Labs, Inc.
 // SPDX-License-Identifier: Apache-2.0
 
+use super::{IDOperation, ObjectChange};
 use crate::base_types::{
     random_object_ref, EpochId, ObjectID, ObjectRef, SequenceNumber, SuiAddress, TransactionDigest,
 };
 use crate::digests::{ObjectDigest, TransactionEventsDigest};
 use crate::effects::{InputSharedObject, TransactionEffectsAPI, UnchangedSharedKind};
-use crate::execution_status::ExecutionStatus;
+use crate::execution_status::{ExecutionFailureStatus, ExecutionStatus};
 use crate::gas::GasCostSummary;
+use crate::move_package::Abort;
 use crate::object::Owner;
+use move_command_line_common::error_bitset::ErrorBitset;
```

Comment:
blech. I hate that this is in move-command-line-common, but nothing to be done about it at least in this PR.